src/com/android/volley/Request.java

```diff
@@ -699,4 +704,17 @@ public void setJoined(boolean joined) {
     public void expireCache() {
         // no - op
     }
+
+    public boolean isExecuting() {
```

Comment:
i'll refactor it separately. Since it would be a larger diff if i find replaced existing usages.